recipes/palikiss/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+make PREFIX=$PREFIX CC=$CC -C Misc/Applications/pAliKiss all
```

Comment:
```suggestion
make -j ${CPU_COUNT} PREFIX=$PREFIX CC=$CC -C Misc/Applications/pAliKiss all
```

Reply:
Thanks @martin-g. Makes total sense to run make in parallel here.